configs/detr/detr_r50_8x4_1x_coco.py

```diff
@@ -0,0 +1,94 @@
+_base_ = [
+    '../_base_/datasets/coco_detection.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+model = dict(
+    type='DETR',
+    pretrained='torchvision://resnet50',
+    backbone=dict(
+        type='ResNet',
+        depth=50,
+        num_stages=4,
+        out_indices=(3, ),
+        frozen_stages=1,
+        norm_cfg=dict(type='BN', requires_grad=True),
+        norm_eval=True,
+        style='pytorch'),
+    bbox_head=dict(
+        type='TransformerHead',
+        num_classes=80,
+        in_channels=2048,
+        num_fcs=2,
+        transformer=dict(
+            type='Transformer',
+            embed_dims=256,
+            num_heads=8,
+            num_encoder_layers=6,
+            num_decoder_layers=6,
+            feedforward_channels=2048,
+            dropout=0.1,
+            act_cfg=dict(type='ReLU', inplace=True),
+            norm_cfg=dict(type='LN'),
+            num_fcs=2,
+            pre_norm=False,
+            return_intermediate_dec=True),
+        position_encoding=dict(
+            type='SinePositionEmbedding', num_feats=128, normalize=True),
+        loss_cls=dict(
+            type='CrossEntropyLoss',
+            bg_cls_weight=0.1,
+            use_sigmoid=False,
+            loss_weight=1.0,
+            class_weight=1.0),
+        loss_bbox=dict(type='L1Loss', loss_weight=5.0),
+        loss_giou=dict(type='GIoULoss', loss_weight=2.0)))
+# training and testing settings
+train_cfg = dict(
+    _delete_=True,
```

Comment:
_delete_ is unnecessary here because the config does not inherit any model config.